nvbench/detail/measure_cold.cu

```diff
@@ -298,6 +341,17 @@ void measure_cold_base::generate_summaries()
     summ.set_string("hide", "Hidden by default.");
   }
 
+  if (!m_sm_clock_rates.empty())
+  {
+    auto &summ = m_state.add_summary("nv/cold/sm_clock_rate/mean");
```

Comment:
Suggestion: Having an additional hidden column with this as a percentage of peak would be useful, too.

Reply:
I'll have to leave this up to you to address as a follow up